lms/templates/problem.html

```diff
@@ -5,13 +5,13 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header" tabindex="-1">
+<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
```

Comment:
Please hook this up to the "scroll to top of problem" method that @alisan617 introduced.